src/entity/posts/utils.ts

```diff
@@ -352,6 +353,15 @@ interface CreateSharePostArgs {
   args: SharePostArgs;
 }
 
+export const determineSharedPostId = (post: Post | SharePost): string => {
+  if (post.type === PostType.Share && !post.title) {
+      return post instanceof SharePost ? post.sharedPostId : post.id;
+    }
+  }
```

Comment:
```suggestion
  if (post.type === PostType.Share && !post.title) {
    return post instanceof SharePost ? post.sharedPostId : post.id;
  }
```
should maybe remove the dangling `}` too 😬